README.md

```diff
@@ -21,8 +21,12 @@ performance optimizations we’ve added along the way. Among those is a standard
 framerate-based renderer, a renderer for high-performance scrollable
 regions which works alongside the main renderer, and mouse support.
 
-To get started, see the tutorial below, the [examples][examples], the
-[docs][docs] and some common [resources](#libraries-we-use-with-bubble-tea).
+## Getting Started
+
+We recommend starting with our [basics tutorial](https://github.com/charmbracelet/bubbletea/tree/master/tutorials/basics) followed by our [commands tutorial](https://github.com/charmbracelet/bubbletea/tree/master/tutorials/commands) to understand what you can do with `tea.Cmd` and how they're used in BubbleTea. 
```

Comment:
With regard to paragraphs let's generally format them so that the README is just as readable in plain text as it is in Glow and on GitHub. This means:

* Wrap at 80 columns
* Generally move links to references in paragraphs to keep long links from interrupting the user’s reading experience.